.github/workflows/ci.yml

```
@@ -35,6 +35,30 @@ jobs:
+
+  shellcheck:
+    name: shellcheck
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v6
+      - name: install
+        run: sudo apt-get update -q && sudo apt-get install -y -q shellcheck
+      # Warning level: an info-level note about a quoted variable in a
+      # bundled installer is not worth a red PR.
+      - name: shellcheck
+        run: shellcheck -S warning $(git ls-files 'internal/recipes/bundled/*.sh' 'internal/recipes/bundled/*/*.sh' 'scripts/*.sh' '.githooks/*')
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2046:warning:1:23: Quote this to prevent word splitting

(shellcheck)